install_config/web_console_customization.adoc

```diff
@@ -338,6 +338,42 @@ angular
 hawtioPluginLoader.addModule('<myExtensionModule>');
 ----
 
+=== Application Launcher
+
+The top navigation bar also contains an optional application launcher for
+linking to other web applications. This dropdown menu is empty by default, but
+when links are added, appears to the left of the help menu in the masthead.
+
+. Create the configuration scripts within a file (for example,
+*_applicationLauncher.js_*):
+
+[source, javascript]
+----
+// Add items to the application launcher dropdown menu.
+window.OPENSHIFT_CONSTANTS.APP_LAUNCHER_NAVIGATION = [{
+  label: "Dashboard",                    // The text label
+  iconClass: "fa fa-dashboard",          // The icon you want to appear
+  href: "http://example.com/dashboard",  // Where to go when this item is clicked
+  tooltip: 'View dashboard'              // Optional tooltip to display on hover
+}, {
+  label: "Manage Account",
+  iconClass: "pficon pficon-user",
+  href: "http://example.com/account",
+  tooltip: "Update email address or password."
+}];
+----
+
+. Save the file and add it to the master configuration at
+*_/etc/origin/master/master-config.yml_*:
```

Comment:
Everywhere in this file, we use `master-config.yml`, but I believe it's `master-config.yaml`. I can push an update.